### PR TITLE
[TM-1104] display map in submissions and applications

### PR DESCRIPTION
**Pull request by cesarLima1**

### CODE REVIEW

File: src/pages/project-pitches/components/tabs/PitchOverviewTab.tsx [hunk 1]
```diff
@@ -32,14 +35,44 @@ const PitchOverviewTab = ({ pitch }: PitchOverviewTabProps) => {
...
-  let projectBoundary: any;
-  try {
-    projectBoundary = JSON.parse(pitch.proj_boundary!);
-  } catch (e) {
-    projectBoundary = undefined;
-  }
+  const setBbboxAndZoom = async () => {
+    if (projectPolygon?.project_polygon?.poly_uuid) {
+      const bbox = await fetchGetV2TerrafundPolygonBboxUuid({
+        pathParams: { uuid: projectPolygon.project_polygon?.poly_uuid ?? "" }
```
Comment (pachonjcl) [inline src/pages/project-pitches/components/tabs/PitchOverviewTab.tsx:58]: you have already checked that poly_uuid is truthy no need for this

File: src/pages/project-pitches/components/tabs/PitchOverviewTab.tsx [hunk 2]
```diff
@@ -32,14 +35,44 @@ const PitchOverviewTab = ({ pitch }: PitchOverviewTabProps) => {
...
+  );
 
-  let projectBoundary: any;
-  try {
-    projectBoundary = JSON.parse(pitch.proj_boundary!);
-  } catch (e) {
-    projectBoundary = undefined;
-  }
+  const setBbboxAndZoom = async () => {
+    if (projectPolygon?.project_polygon?.poly_uuid) {
```
Comment (pachonjcl) [inline src/pages/project-pitches/components/tabs/PitchOverviewTab.tsx:56]: you have already checked that poly_uuid is truthy no need for this

File: src/pages/project-pitches/components/tabs/PitchOverviewTab.tsx [hunk 3]
```diff
@@ -32,14 +35,44 @@ const PitchOverviewTab = ({ pitch }: PitchOverviewTabProps) => {
...
+  };
+
+  useEffect(() => {
+    const getDataProjectPolygon = async () => {
+      if (!projectPolygon?.project_polygon) {
+        setPolygonDataMap({ [FORM_POLYGONS]: [] });
+      } else {
+        setBbboxAndZoom();
+        setPolygonDataMap({ [FORM_POLYGONS]: [projectPolygon?.project_polygon?.poly_uuid] });
+      }
```
Comment (pachonjcl) [inline src/pages/project-pitches/components/tabs/PitchOverviewTab.tsx:72]: nitpick: can you handle the true case first?

**Review by pachonjcl — CHANGES_REQUESTED** [review]

File: src/pages/project-pitches/components/tabs/PitchOverviewTab.tsx (hunk 2 above)
Comment (cesarLima1) [reply src/pages/project-pitches/components/tabs/PitchOverviewTab.tsx:56]: already changed

File: src/pages/project-pitches/components/tabs/PitchOverviewTab.tsx (hunk 3 above)
Comment (cesarLima1) [reply src/pages/project-pitches/components/tabs/PitchOverviewTab.tsx]: great, done

**Review by pachonjcl — APPROVED** [review]

**Pull request merged**